services/121-service/src/metrics/metrics.service.ts

```diff
@@ -747,7 +749,7 @@ export class MetricsService {
       newPeople,
       registeredPeople,
       totalBudget,
-      spentMoney,
```

Comment:
@jannisvisser Looking good. Should we also make an api test here to cover this. Or do you think meh it was never there so let's leave it out of scope

Reply:
see comments in [evaluate task](https://dev.azure.com/redcrossnl/121%20Platform/_sprints/taskboard/121%20Development%20Team/121%20Platform/Sprint%20158?workitem=36935)